src/commands/transaction/pullfunds.ts

```diff
@@ -32,26 +32,13 @@ import {
     TransferTransaction,
     UInt64,
 } from 'nem2-sdk';
-import {AddressValidator} from '../../address.validator';
+import {AddressValidator} from '../../validators/address.validator';
```

Comment:
Is not strictly part of this PR, but could you get rid of lines 102-109 as you are using the address validator.